src/Utilities/Workloads.js

```diff
@@ -147,29 +147,22 @@ export const pruneWorkloadsRulesFilters = (localFilters, filterCategories) => {
 };
 
 export const switchSort = (sortIndex, item) => {
-  const rule = item[0]?.rule; // Using optional chaining to check if 'rule' is defined
+  const rule = item[0].rule;
   switch (sortIndex) {
     case 1:
       return rule?.details || '';
     case 2:
-      return rule?.total_risk || [];
+      return rule?.total_risk || '';
     case 3:
       return rule?.objects?.length || 0;
     case 4:
       return rule?.modified || '';
```

Comment:
I am not sure it makes sense to use default values at all here. All of the fields are expected to be present in the response, is it correct? In that case we have to remove the optional chaining and also the default values. Anything odd happening: we should have a proper error, not a value which is guessed and replaced as a placeholder.

```
    case 1:
      return rule.details;
    case 2:
      return rule.total_risk;
    case 3:
      return rule.objects.length;
    case 4:
      return rule.modified;
```

Reply:
Ok, replacing that